pages/expenses.js

```diff
@@ -89,10 +131,128 @@ export default function Expenses({ expenses }) {
       <Header>Ausgaben</Header>
       <main>
         <ContentWrapper>
-          <Scatter options={plotOptions} data={plotData} />
+          <GraphWrapper>
+            {Object.keys(chartData).length !== 0 && (
+              <Scatter options={chartOptions} data={chartData} />
+            )}
+          </GraphWrapper>
+          {!isShowForm ? (
+            <OpenFormButton
+              aria-label="Öffne Formular für Ausgaben"
+              onClick={() => setIsShowForm(true)}
+            >
+              <MoneyAddIcon
+                width={40}
+                height={40}
+                fillColor={"var(--background-primary)"}
+                alt="Geldstapel-Icon"
+              />
+            </OpenFormButton>
+          ) : (
+            <Modal onCloseModal={() => setIsShowForm(false)}>
+              <StyledForm onSubmit={handleSubmit}>
+                <InputWrapper>
+                  <StyledLabel htmlFor="amount">
+                    Ausgaben hinzufügen
+                  </StyledLabel>
+                  <StyledInput
+                    type="number"
+                    min={0.01}
+                    max={500}
+                    step={0.01}
+                    name="amount"
+                    id="amount"
+                    placeholder="0.00 €"
+                    required
+                  />
+                </InputWrapper>
+                <InputWrapper>
+                  <StyledLabel htmlFor="date">Datum</StyledLabel>
+                  <StyledInput
+                    type="date"
+                    min="2022-01-01"
+                    max="2099-12-12"
```

Comment:
Suggestion: take today as max value, as you don't have expenditures done today for tomorrow I guess. Unless it is not for tracking purposes, but for planning instead.